src/js-context.ts

```diff
@@ -63,6 +63,14 @@ export default class JSContext {
     return methodNames;
   }
 
+  getClassName(): string | null {
```

Comment:
I checked, and we don't yet have tests for `js-context`, so in light that today's my last day before vacation, I'll leave this for another day 😉